python/llm/example/transformers/native_int4_pipeline_readme.md

```diff
@@ -0,0 +1,127 @@
+# INT4 Inference Pipeline for Large Language Model using BigDL-LLM Native API
+
+In this example, we show a pipeline to convert a large language model to low precision (INT4), and then conduct inference on the converted INT4 model, using BigDL-LLM native API.
```

Comment:
In this example, we show a pipeline to convert a large language model to bigdl-llm native INT4 format, and then run inference on the converted INT4 model.